compiler/src/test/kotlin/io/spine/protodata/PipelineTest.kt

```diff
@@ -203,6 +205,47 @@ class `'Pipeline' should` {
             .doesNotContain(GenericInsertionPoint.OUTSIDE_FILE.codeLine)
     }
 
+    @Test
+    fun `write code into different destination`() {
+        val destination = Files.createTempDirectory("destination")
```

Comment:
Let's extract this directory creation into a method and move the comment there — instead of having it twice (and in future, maybe even more times) in the codebase.